internal/commands/push.go

```diff
@@ -136,7 +137,7 @@ func pushInvocationImage(dockerCli command.Cli, retag retagResult) error {
 }
 
 func pushBundle(dockerCli command.Cli, opts pushOptions, bndl *bundle.Bundle, retag retagResult) error {
-	insecureRegistries, err := insecureRegistriesFromEngine(dockerCli)
+	insecureRegistries, err := internal.InsecureRegistriesFromEngine(dockerCli)
```

Comment:
Maybe `pushBundle` could be in `cnab` package, as `PullBundle` is? And then move `internal.InsecureRegistriesFromEngine` to `cnab` package too?

Reply:
We'll do it in a followup, this PR is already big as it is